src/tempShipment/controller.js

```diff
@@ -33,3 +35,19 @@ async function createOne(req, res) {
     "creating temp shipment"
   );
 }
+
+/**
+ * Create a new temp shipment given a manifest
+ */
+async function deleteOne(req, res) {
+  ExpressHandler(
+    async () => {
+      const { tsid } = req.params;
+
+      // delete temp shipment
+      await TempShipment.deleteOne({ _id: tsid });
```

Comment:
should we throw an error is this tsid doesnt exist?

Reply:
Resolved